publikationsleitlinien.md

```diff
@@ -90,6 +91,11 @@ Werte mit Daten-Format `Uhrzeit` geben wir dem [internationalen Standard ISO 860
 ##### 5. URL 
 Werte mit Daten-Format `URL` schreiben wir standardmässig aufrufbar aus im Format `https://...`. <br>
 
+##### 6. Geografische Lage 
+Werte mit Daten-Format `Geokoordinaten` geben wir dem [globalen geodätischen Referenzsystem WGS 84](https://www.w3.org/TR/sdw-bp/) gemäss an als:
+- `latitude` (z.B. `47.3786314`) und
+- `longitude` (z.B. `8.54110776`).
```

Comment:
Ich vermisse einen Hinweis auf die Präzision der Werte. Das Beispiel hat 8 Kommastellen, je nach Längengrad bewegen wir uns da im Millimeterbereich. Zu viele Kommastellen stellen IMHO eine Präzision dar, die weder gebraucht noch eingehalten werden kann.

Ich habe keinen konkreten Vorschlag zur Anzahl Kommastellen, aber es sollte zumindest als Empfehlung hier stehen.

Reply:
Hier gibt es Überlegungen dazu: https://wiki.openstreetmap.org/wiki/Precision_of_coordinates#Conversion_to_decimal Vielleicht kann man in den Empfehlungen aufnehmen, dass die Anzahl Dezimalstellen der gewünschten Präzision angepasst werden soll. Empfehlung ist die Tabelle Reasonable Display Accuracy.